### ISSUE TITLE
Crash when updating from 2.4.0 to 2.4.1

### ISSUE THREAD

**Issue by Jeff11** [pusher/pusher-websocket-java#338]
When updating from 2.4.0 to 2.4.1 I immediately got this crash.
```
FATAL EXCEPTION: pusher-java-client eventQueue
  Process: my.app.package, PID: 29420
  java.lang.UnsupportedOperationException: JsonObject
    at com.google.gson.JsonElement.getAsString(JsonElement.java:179)
    at com.pusher.client.channel.PusherEvent.fromJson(PusherEvent.java:88)
    at com.pusher.client.connection.websocket.WebSocketConnection.lambda$onMessage$5$com-pusher-client-connection-websocket-WebSocketConnection(WebSocketConnection.java:212)
    at com.pusher.client.connection.websocket.WebSocketConnection$$ExternalSyntheticLambda3.run(Unknown Source:4)
    at com.pusher.client.util.Factory.lambda$queueOnEventThread$0(Factory.java:140)
    at com.pusher.client.util.Factory$$ExternalSyntheticLambda0.run(Unknown Source:2)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
    at java.lang.Thread.run(Thread.java:1012)
```

**Comment by fbenevides** [comment]
Hi @Jeff11, we're investigating this issue. Would you have more information on this? Things like the Android version (or another platform) would greatly help us.

Thanks!

**Comment by Jeff11** [comment]
It happened on Android 13.

**Comment by proggen-com** [comment]
I think this PR fixes the issue: https://github.com/pusher/pusher-websocket-java/pull/339

**Comment by benjamin-tang-pusher** [comment]
I can't reproduce this on the new 2.4.2 so it was likely fixed by #339
